roles/matrix-ma1sd/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+# Doing `|from_yaml` when the extension contains nothing yields an empty string ("").
+# We need to ensure it's a dictionary or `|combine` (when building `matrix_ma1sd_configuration`) will fail later.
+matrix_ma1sd_configuration_extension: "{{ matrix_ma1sd_configuration_extension_yaml|from_yaml if matrix_ma1sd_configuration_extension_yaml|from_yaml else {} }}"
```

Comment:
missing new line

Reply:
uhmm.. thanks for the review.. this is the same as in the other roles though, isn't it?

Reply:
git cares about this kinds of stuff and can generate stupid diffs if there is no new line at the end.

Reply:
That may well be; however none of the other `roles/*/var/main.yml`s have a trailing new line.

Reply:
Oh just learned this is about an invisible `0A` byte at the end.. thanks for insisting on that xD